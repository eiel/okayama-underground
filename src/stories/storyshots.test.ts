// @ts-ignore
// tslint:disable-next-line:no-implicit-dependencies
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots();
