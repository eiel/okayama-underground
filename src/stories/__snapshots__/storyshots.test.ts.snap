// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots OugH1 with text 1`] = `
<h1
  className="h1"
>
  Hello World
</h1>
`;
